code/modules/client/preference_setup/loadout/loadout.dm

```diff
@@ -86,6 +96,9 @@ var/list/gear_datums = list()
 	if(!islist(pref.gear))
 		pref.gear = list()
 
+	if(!islist(pref.gear_list))
```

Comment:
swap this with the check on line 96. gear_list should be checked first, then we should try to assign it:
```dm
  if (!isnull(pref.gear_slot) && islist(pref.gear_list["[pref.gear_slot]"]))
    pref.gear = pref.gear_list["[gear_slot]"]
  else
    // old format, try to recover it.
    if (!islist(pref.gear_list["1"]))
      pref.gear = pref.gear_list.Copy()
      pref.gear_list = list("1" = pref.gear)
      pref.gear_slot = 1
    else
      pref.gear = list()
      pref.gear_list = list("1" = pref.gear)
      pref.gear_slot = 1
```